modemController.go

```diff
@@ -33,14 +33,14 @@ type ModemController struct {
 	Modem             *Modem
 	ModemsConfig      []ModemConfig
 	TestHosts         []string
-	TestInterval      int
+	TestInterval      time.Duration
```

Comment:
Does this work? For a TOML file I had to wrap it in a custom type (as per those links I provided earlier). Great if it works out of the box though.